src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/events/handlers/PrisonerVideoAppointmentCancelledEventHandlerTest.kt

```diff
@@ -71,244 +60,9 @@ class PrisonerVideoAppointmentCancelledEventHandlerTest {
-        cancellationEvent(
-          prisonCode = WANDSWORTH,
-          prisonerNumber = "ABC345",
-          start = tomorrow().atStartOfDay(),
-          eventType = "VLB",
-        ),
-      )
-
-      inOrder(videoAppointmentRepository, prisonAppointmentRepository, videoBookingRepository, bookingHistoryService) {
-        verify(videoAppointmentRepository).findActiveVideoAppointments(
-          WANDSWORTH,
-          "ABC345",
-          appointmentDate = tomorrow(),
-          startTime = LocalTime.MIDNIGHT,
-        )
-
-        verify(prisonAppointmentRepository).deleteById(courtAppointment.prisonAppointmentId)
-        verify(prisonAppointmentRepository).flush()
-        verify(videoBookingRepository).findById(courtBooking.videoBookingId)
-        verify(bookingHistoryService).createBookingHistory(HistoryType.AMEND, courtBooking)
-      }
-
-      verifyNoInteractions(bookingFacade)
-    }
-
-    @Test
-    fun `should attempt to cancel a probation booking removed in NOMIS`() {
-      whenever(activitiesAppointmentsClient.isAppointmentsRolledOutAt(any())) doReturn false
-      val probationBooking = probationBooking().withProbationPrisonAppointment(prisonerNumber = "DEF345", prisonCode = PENTONVILLE)
-      val probationAppointment = videoAppointment(probationBooking, probationBooking.appointments().single())
-
-      whenever(
-        videoAppointmentRepository.findActiveVideoAppointments(
-          prisonCode = PENTONVILLE,
-          prisonerNumber = "DEF345",
-          appointmentDate = tomorrow(),
-          startTime = LocalTime.MIDNIGHT,
-        ),
-      ) doReturn listOf(probationAppointment)
-
-      handler.handle(
-        cancellationEvent(
-          prisonCode = PENTONVILLE,
-          prisonerNumber = "DEF345",
-          start = tomorrow().atStartOfDay(),
-          eventType = "VLB",
-        ),
-      )
-
-      inOrder(videoAppointmentRepository, bookingFacade) {
-        verify(videoAppointmentRepository).findActiveVideoAppointments(
-          PENTONVILLE,
-          "DEF345",
-          appointmentDate = tomorrow(),
-          startTime = LocalTime.MIDNIGHT,
-        )
-        verify(bookingFacade).cancel(probationBooking.videoBookingId, SERVICE_USER)
-      }
-    }
-
-    @Test
-    fun `should not cancel a (VLPM) probation booking removed in NOMIS`() {
-      whenever(activitiesAppointmentsClient.isAppointmentsRolledOutAt(any())) doReturn false
-      val probationBooking = probationBooking().withProbationPrisonAppointment(prisonerNumber = "DEF345", prisonCode = PENTONVILLE)
-      val probationAppointment = videoAppointment(probationBooking, probationBooking.appointments().single())
-
-      whenever(
-        videoAppointmentRepository.findActiveVideoAppointments(
-          prisonCode = PENTONVILLE,
-          prisonerNumber = "DEF345",
-          appointmentDate = tomorrow(),
-          startTime = LocalTime.MIDNIGHT,
-        ),
-      ) doReturn listOf(probationAppointment)
-
-      handler.handle(
```

Comment:
A lot of tests removed here - are they all replicated in other places or have a substitute now the switch has been removed? i.e. Are we losing anything useful?

Reply:
No we are all good.  The tests removed are those where the VLPM feature would be off/false which is no longer applicable since it has been removed.  Tests to cover where it was on is now the default.